test_app/lib/screens/test_buttons/conference_service_test_buttons.dart

```diff
@@ -274,19 +276,71 @@ class _ConferenceServiceTestButtonsState
   Future<void> getLocalStats() async {
     try {
       var rtcStatsTypes =
-      await _dolbyioCommsSdkFlutterPlugin.conference.getLocalStats();
+          await _dolbyioCommsSdkFlutterPlugin.conference.getLocalStats();
       if (!mounted) return;
       showResultDialog(context, 'Success', jsonEncode(rtcStatsTypes));
     } catch (error) {
       if (!mounted) return;
       showResultDialog(context, 'Error', error.toString());
     }
   }
-  
-  Future<void> setVideoForwarding(BuildContext context) async {
+
+  Future<void> setVideoForwarding(BuildContext setVideoContext) async {
+    return await showDialog(
+      context: setVideoContext,
+      barrierDismissible: false,
+      builder: (BuildContext setVideoForwardingContext) {
+        return AlertDialog(
+          title: const Text('Set Video Forwarding'),
+          content: Column(mainAxisSize: MainAxisSize.min, children: [
+            Consumer<SpatialValuesModel>(
+                builder: (context, spatialValuesModel, child) {
+              return Column(
+                children: [
+                  TextField(
+                    controller: _videoForwardingXTextController
+                      ..text = spatialValuesModel.maxVideoForwarding.toString(),
+                    keyboardType: TextInputType.number,
+                    inputFormatters: <TextInputFormatter>[
+                      FilteringTextInputFormatter.digitsOnly
+                    ],
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: [
+                      TextButton(
+                        child: const Text('OK',
+                            style: TextStyle(color: Colors.deepPurple)),
+                        onPressed: () {
+                          spatialValuesModel.setMaxVideoForwarding(
+                              int.parse(_videoForwardingXTextController.text));
+                          setForwarding(
+                              int.parse(_videoForwardingXTextController.text));
+                          Navigator.of(setVideoForwardingContext).pop();
+                        },
+                      ),
+                      TextButton(
+                        child: const Text('CANCEL',
+                            style: TextStyle(color: Colors.deepPurple)),
+                        onPressed: () {
+                          Navigator.of(setVideoForwardingContext).pop();
+                        },
+                      )
+                    ],
+                  ),
+                ],
+              );
+            })
+          ]),
+        );
+      },
+    );
+  }
```

Comment:
Is this part of the fix? And this question is also applicable to the rest of the changes apart form those in session_service.dart file.